scripts/yarn/vets-website-assets.js

```diff
@@ -83,25 +83,13 @@ async function downloadFromLiveBucket(buildtype) {
 }
 
 // Gather assets that are expected by the compiled files but not included in the bucket (because content-build would source them).
-// These are font files, icons, and other assorted images.
+// These are primarily images.
```

Comment:
Font files are now provided by vets-website; font-awesome icons are no longer in use. We only need images.